docs/manual/working/scalaGuide/main/json/ScalaJson.md

```diff
@@ -38,24 +38,24 @@ The [`play.api.libs.json`](api/libs/json/) package contains data structures for
 
 The package provides the following types:
 
-### [`JsValue`](api/libs/json/JsValue.html)
+### [`JsValue`](api/scala/play/api/libs/jsonJsValue.html)
 
 This is a trait representing any JSON value. The JSON library has a case class extending `JsValue` to represent each valid JSON type:
 
-- [`JsString`](api/libs/json/JsString.html)
-- [`JsNumber`](api/libs/json/JsNumber.html)
-- [`JsBoolean`](api/libs/json/JsBoolean.html)
-- [`JsObject`](api/libs/json/JsObject.html)
-- [`JsArray`](api/libs/json/JsArray.html)
-- [`JsNull`](api/libs/json/JsNull$.html)
+- [`JsString`](api/scala/play/api/libs/jsonJsString.html)
+- [`JsNumber`](api/scala/play/api/libs/jsonJsNumber.html)
+- [`JsBoolean`](api/scala/play/api/libs/jsonJsBoolean.html)
+- [`JsObject`](api/scala/play/api/libs/jsonJsObject.html)
+- [`JsArray`](api/scala/play/api/libs/jsonJsArray.html)
+- [`JsNull`](api/scala/play/api/libs/jsonJsNull$.html)
 
 Using the various `JsValue` types, you can construct a representation of any JSON structure.
 
-### [`Json`](api/libs/json/Json$.html)
+### [`Json`](api/scala/play/api/libs/jsonJson$.html)
```

Comment:
I think you forgot a `/`: `api/scala/play/api/libs/json/Json$.html` (same issue elsewhere in this file).

Reply:
Sorry! I`ll fix that.